src/js/jsx/sections/export/ExportAllPanel.jsx

```diff
@@ -0,0 +1,226 @@
+                    return exportActions.exportAllAssets(document);
+                });
+            }.bind(this));
+        },
+
+        render: function () {
+            var document = this.state.document,
+                layerExportsMap = this.state.documentExports && this.state.documentExports.layerExportsMap,
+                freshState = this.state.fresh;
+
+            if (!document || !layerExportsMap) {
+                return null;
+            }
+
+            // Iterate over all configured assets, build the individual components,
+            // and separate into separate lists (artboards vs. non-artboards)
+            var layerExportComponents = [],
+                artboardExportComponents = [];
+            if (layerExportsMap && layerExportsMap.size > 0) {
+                layerExportsMap.forEach(function (layerExports, key) {
+                    var layer = document.layers.byID(key);
+
+                    if (layer && layerExports && layerExports.size > 0) {
+                        var layerComponent = (
+                            <PanelLayer
+                                document={document}
+                                layer={layer}
+                                layerExports={layerExports}
+                                freshState={freshState}
+                                key={"layer-" + layer.id} />
+                        );
+
+                        if (layer.isArtboard) {
+                            artboardExportComponents.push(layerComponent);
+                        } else {
+                            layerExportComponents.push(layerComponent);
+                        }
+                    }
+                });
+            }
+
+            return (
+                <div className="exports-panel__container">
+                    <TitleHeader
+                        title={strings.TITLE_EXPORT} />
+                    <div className="exports-panel__two-column">
+                        <div className="exports-panel__asset-list__container">
+                            <TitleHeader
+                                title="Artboards" />
+                            <div className="exports-panel__asset-list__list">
+                                {artboardExportComponents}
+                            </div>
+                        </div>
+                        <div className="exports-panel__asset-list__container">
+                            <TitleHeader
+                                title="Layers" />
+                            <div className="exports-panel__asset-list__list">
+                                {layerExportComponents}
+                            </div>
+                        </div>
+                    </div>
+                    <hr />
+                    <div className="exports-panel__button-group">
+                        <Button
+                            onClick={this.props.dismissDialog}>
+                            Cancel
```

Comment:
Should be localized. (And, yes, we should do this ASAP so that the loc team can get started even before the UI is final.)